src/runtime_src/core/common/ishim.h

```diff
@@ -391,38 +391,39 @@ struct shim : public DeviceType
   }
 
   void
-  copy_bo(xclBufferHandle dst, xclBufferHandle src, size_t size, size_t dst_offset, size_t src_offset) override
+  copy_bo(xrt_buffer_handle dst, xrt_buffer_handle src, size_t size, size_t dst_offset, size_t src_offset) override
   {
-    if (auto err = xclCopyBO(DeviceType::get_device_handle(), dst, src, size, dst_offset, src_offset))
+    if (auto err = xclCopyBO(DeviceType::get_device_handle(),
```

Comment:
There is a lot going on in this line, please consider reformatting it.
```suggestion
    auto err = xclCopyBO(DeviceType::get_device_handle(),to_xclBufferHandle(dst), to_xclBufferHandle(src), size, dst_offset, src_offset);
    if (err)
```

Reply:
Broke it into two lines.